graftm/hmmer.py

```diff
@@ -358,7 +365,7 @@ def _check_euk_contamination(self, hmm_hit_tables):
         
         euk_reads = reads_with_better_euk_hit + reads_unique_to_eukaryotes
         
-        return euk_reads
+        return set(euk_reads)
```

Comment:
I think at one point you asked me why I used a set. Its because I use this set to filter for non euk reads downstream. So I used a set because they're faster for membership testing.